website/docs/intro/install/docker.mdx

```diff
@@ -0,0 +1,47 @@
+---
+sidebar_position: 99
+sidebar_label: Docker Image
+description: |-
+  Use official Docker Image available on GitHub Container Registry.
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import DockerPullScript from '!!raw-loader!./docker-pull.sh'
+import DockerRunScript from '!!raw-loader!./docker-run.sh'
+
+# Use OpenTofu as Docker Image
+
+OpenTofu is available as [Docker images](https://github.com/opentofu/opentofu/pkgs/container/opentofu),
+and distributed via public GitHub Packages registry.
+
+## Versions
+
+Images are hosted as packages in opentofy GitHub organization. See the list
+of available versions [here](https://github.com/opentofu/opentofu/pkgs/container/opentofu/versions?filters%5Bversion_type%5D=tagged).
+
+These multi-arch tags are available:
+
+- `latest`: latest overall version of OpenTofu,
+- `Major`: a specific major version of OpenTofu,
+- `Major`.`Minor`: a specific minor version of OpenTofu,
+- `Major`.`Minor`.`Patch`: a specific patch version of OpenTofu.
+
+To pull platform-specific images (`amd64`, `arm`, `arm64`, `386`) use:
+
+- `<Version>`-`<Platform>`: a platform specific version of OpenTofu.
+
+## Usage
+
+To be able to pull the image generate a PAT with scope `packages:read` and then
+use it to [login][ghcr-login] to GitHub Packages registry.
```

Comment:
This is incorrect, you can pull without logging in, but rate limits may apply (see suggestion below).

```suggestion
```